### DISCUSSION TITLE
Datalogger for Windows

### DISCUSSION THREAD

**Discussion by pierregraciano** [micropython/micropython#14277] [Hardware & Peripherals]
Hi guys. Im looking for a datalogger to use micropython on Thonny IDE, that is free, no module or SD card and compatible with Windows. I saw some videos using IFTTT but unfortunately this website is paid. The datalogger needs to share live info to some sheet or txt file.
Can anyone tell me if there is possible? Thank you!

**Comment by sosi-deadeye** [comment]
You could use Python on Windows to do this task.

There are many different possible solutions.
I would use a socket to log data. The Program on Windows could log the data to a sqlite3 database.

**Comment by KnorkFox** [comment]
@pierregraciano  your wish is a little bit old, but if you still want an answer, i try my best.

For a beginner, is this book from Raspberry Pi Press:
"Get started with micropython on Raspberry Pi Pico"
On page 104, is a datalogger project to code.
